.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: xb7mdkOt7HvqNT5kHZ0ge3YV0jRJgJfcw
```

Comment:
You'll need to have a single commit for gitignores so subsequent commits properly ignore the right paths. To correct/redo this existing commit, you can run `git rm -r --cached .`

This untracks all files/paths listed in gitignore (or you can run `git rm -r --cached .coveralls.yml` to be more selective). Then `git add .` and `git commit --amend --no-edit`.